scripts/limonade/src/index.ts

```diff
@@ -197,6 +267,71 @@ async function run() {
     }, argv.solanaWebsocketTimeout);
   });
 
+  if (priceConfigs.length > 0) {
+    const hermesClient = new HermesClient(argv.hermesEndpoint, {});
+
+    const eventSource = await hermesClient.getPriceUpdatesStream(
+      priceConfigs.map((priceConfig) => priceConfig.pythFeedId),
+      {
+        encoding: "hex",
+        parsed: true,
+        ignoreInvalidPriceIds: true,
+      }
+    );
+
+    eventSource.onerror = (event: Event) => {
+      console.error("Hermes streaming error", event);
+    };
+
+    /// Await for the first message before continuing
```

Comment:
I think we use double slash in most of the places

Reply:
done